src/main/java/org/broadinstitute/consent/http/resources/DraftResource.java

```diff
@@ -119,6 +120,23 @@ public Response updateDraft(@Auth AuthUser authUser, @PathParam("draftUUID") Str
     }
   }
 
+  @PATCH
+  @Produces({MediaType.APPLICATION_JSON})
+  @Consumes({MediaType.TEXT_PLAIN})
+  @Path("/v1/{draftUUID}")
+  @RolesAllowed({ADMIN, DATASUBMITTER})
+  public Response patchDraftName(@Auth AuthUser authUser, @PathParam("draftUUID") String draftUUID, String name) {
+    try {
+      User user = userService.findUserByEmail(authUser.getEmail());
+      DraftInterface draft = draftService.getAuthorizedDraft(validateUUID(draftUUID), user);
```

Comment:
General comment here, no action needed unless we do some refactoring. In `draftService.getAuthorizedDraft`, we check for user permissions on the draft. In general, we've been applying ACL-like conditions here in the resource layer, not in the business layer.

Reply:
I've seen different approaches to this over time and can certainly incorporate that in the future.